Index: LinkDbReader.java
===================================================================
--- LinkDbReader.java	(revision 1147712)
+++ LinkDbReader.java	(working copy)
@@ -123,7 +123,7 @@
   
   public int run(String[] args) throws Exception {
     if (args.length < 2) {
-      System.err.println("Usage: LinkDbReader <linkdb> {-dump <out_dir> | -url <url>)");
+      System.err.println("Usage: LinkDbReader <linkdb> (-dump <out_dir> | -url <url>)");
       System.err.println("\t-dump <out_dir>\tdump whole link db to a text file in <out_dir>");
       System.err.println("\t-url <url>\tprint information about <url> to System.out");
       return -1;
